xsct.c

```diff
@@ -218,10 +220,32 @@ int main(int argc, char **argv)
         }
         else
         {
-            temp = (temp == 0) ? TEMPERATURE_NORM : temp;
-
-            for (screen = 0; screen < screens; screen++)
-                sct_for_screen(dpy, screen, temp, fdebug);
+            temp = ((temp == 0) && (fdelta == 0)) ? TEMPERATURE_NORM : temp;
+            if (fdelta == 0)
+            {
+                if (temp < TEMPERATURE_ZERO)
```

Comment:
```c
            if (fdelta == 0)
            {
                if (temp == 0)
                {
                    temp = TEMPERATURE_NORM;
                }
                else if (temp < TEMPERATURE_ZERO)
```

Reply:
Hi @faf0 .

Not. I want you to make your own using mine as a template. Too many fixes.

Reply:
implemented here: https://github.com/faf0/sct/pull/18